ScanbotSDKExamples/NET/ClassicComponent.iOS/ViewControllers/MainViewController.cs

```diff
@@ -0,0 +1,438 @@
+            .Select(id => SBSDKUIPageFileStorage.DefaultStorage.ImageURLWithPageFileID(id, SBSDKUIPageFileType.Original))
+            .ToArray();
+                var ocrConfiguration = new OCRDetectionConfiguration
+                {
+                    RecognitionMode = ScanbotSDK.iOS.SBSDKOpticalCharacterRecognitionMode.Legacy,
+                    // Languages property is only used in SBSDKOpticalCharacterRecognitionMode.Legacy
+                    Langauges = SBSDK.InstalledLanguages,
+                    InputImageUrls = inputImages,
+                    ShouldGeneratePDF = false,
+                };
+                var result = await SBSDK.PerformOCR(ocrConfiguration);
+                IsBusy = false;
+                Utilities.ShowMessage("OCR Text", result.RecognizedText);
+            });
+        }
+
+        private void CreatePDF()
+        {
+            if (!CheckScanbotSDKLicense()) { return; }
+            if (!CheckOriginalImageUrl()) { return; }
+
+            Task.Run(async () =>
+            {
+                IsBusy = true;
+                Debug.WriteLine("Creating PDF file ...");
+                var images = SBSDKUIPageFileStorage.DefaultStorage.AllPageFileIDs
+                            .Select(id => SBSDKUIPageFileStorage.DefaultStorage.ImageURLWithPageFileID(id, SBSDKUIPageFileType.Original))
+                            .ToArray();
+                var pdfOutputFileUrl = Utilities.GenerateRandomFileUrlInDemoTempStorage(".pdf");
+
+                await SBSDK.CreatePDF(new CreatePDFConfiguration { InputImageUrls = images, OutputUrl = pdfOutputFileUrl, PageSize = ScanbotSDK.MAUI.Constants.PDFPageSize.A4 });
+                Debug.WriteLine("PDF file created: " + pdfOutputFileUrl);
+                IsBusy = false;
+                Utilities.ShowMessage("PDF file created", "" + pdfOutputFileUrl);
+            });
+        }
+
+        private void CreateTIFF()
+        {
+            if (!CheckScanbotSDKLicense()) { return; }
+            if (!CheckOriginalImageUrl()) { return; }
+            IsBusy = true;
+            Debug.WriteLine("Creating TIFF file ...");
+
+            var inputUrls = SBSDKUIPageFileStorage.DefaultStorage.AllPageFileIDs
+                        .Select(id => SBSDKUIPageFileStorage.DefaultStorage.ImageURLWithPageFileID(id, SBSDKUIPageFileType.Original))
+                        .ToArray();
+
+            var tiffOutputFileUrl = Utilities.GenerateRandomFileUrlInDemoTempStorage(".tiff");
+            //var options = new SBSDKTIFFImageWriterParameters { Binarize = true, Compression = SBSDKTIFFImageWriterCompressionOptions.Ccittfax4, Dpi = 250 };
+
+
+            var options = new TiffOptions { OneBitEncoded = true, Compression = TiffCompressionOptions.CompressionCcittfax4, Dpi = 250 };
+
+            bool success = SBSDK.WriteTiff(inputUrls, tiffOutputFileUrl, options);
+
+
+            // TODO figure out if this is still needed
```

Comment:
@scan-mayank, I think we should be as close to native as possible in this example. 

If we do want to make a convenience method for writing tiffs, I would suggest that we rather make WriteTiff an extension method that can be used with DefaultImageStoreEncrypter and which does some of the extra configuration for the user. At least then, we are not hiding the iOS-specific details from the consumer and they can see it is an extension method to help with usability.

Reply:
Implemented the Native code completely in the separate DocumentUtilities.cs file. And used it in our ViewController.